### ISSUE TITLE
File list during submission: Stage 6

### ISSUE THREAD

**Issue by twakeford** [ubiquitypress/rua#336]
Stage 6 in the submission process lists the files that have been uploaded.

It needs to remain in the order that they were uploaded but currently does not do this. In the below screen shot, the files were uploaded in the following order, which has now changed:
Toc; Acknowledgements; Contributor bio; Introduction; and then the chapters in order
![screen shot 2016-03-14 at 17 39 20](https://cloud.githubusercontent.com/assets/15159847/13753750/36b172b8-ea0c-11e5-97b3-b351d9747f24.png)

**Comment by ajrbyers** [comment]
This will be interesting. We need to get the order the files are sent for upload rather than the order their uploads are completed.

**Comment by IoannisCleary** [comment]
trying to increment file_sequence causes a 500 INTERNAL SERVER ERROR when you use the upload all button. some file sequences will be the same.
![screenshot from 2016-03-16 10-13-00](https://cloud.githubusercontent.com/assets/5523178/13808864/104eb144-eb60-11e5-8900-712d835fdd81.jpg)

need to find a thread safe way to upload files or allow the user to reorder the files and save the sequence

**Comment by IoannisCleary** [comment]
@ajrbyers  @twakeford  it seems that even if you add file sequence that is incremented, you won't get the desired result ( files uploaded/displayed in order 3 - 12 ). so it looks like you would have the same result when  you sort by primary key

**Comment by ajrbyers** [comment]
Yeah we need to give the order they are displayed on the front end and post that. Leave this one for me as it's probably too complex front end stuff.

**Comment by IoannisCleary** [comment]
@ajrbyers  Okay
